convert/gitlab/testdata/templates/cache.yaml.golden

```diff
@@ -0,0 +1,11 @@
+stages:
```

Comment:
Looks like this should file should be this
```yaml
stages:
- name: build
  spec:
    cache:
      enabled: true
      key: binaries-cache
      paths:
      - binaries/*.apk
      - .config
    steps:
    - name: rspec
      spec:
        run: echo "This job uses a cache."
      type: script
  type: ci
version: 1
```

Reply:
changed.